content/docs/examples/advanced-gateways/egress-gateway/index.md

```diff
@@ -842,7 +666,7 @@ to it the [sleep]({{< github_tree >}}/samples/sleep) sample.
     cluster.outbound|443||edition.cnn.com.upstream_cx_total: 2
     {{< /text >}}
 
-### Cleanup of the network policies
+### Cleanup network policies
```

Comment:
So is "cleanup" here used as a verb? "To cleanup network policies"? If it is noun, should it be "Network policies cleanup"?

Reply:
I think it's clear as is. It basically means cleanup for the above network policies section